test/test_object_storage_sftp.py

```diff
@@ -0,0 +1,55 @@
+"""Copyright (c) 2022 Aiven, Helsinki, Finland. https://aiven.io/"""
+from io import BytesIO
+from rohmu.object_storage.sftp import SFTPTransfer
+from tempfile import NamedTemporaryFile
+from unittest.mock import MagicMock, patch
+
+
+def test_store_file_from_disk():
+    notifier = MagicMock()
+    with patch("paramiko.Transport") as _, patch("paramiko.SFTPClient") as sftp_client:
+        client = MagicMock()
+        sftp_client.from_transport.return_value = client
+        transfer = SFTPTransfer(
+            server="sftp.example.com",
+            port=2222,
+            username="testuser",
+            password="testpass",
+            notifier=notifier,
+        )
+        test_data = b"test-data"
+        with NamedTemporaryFile() as tmpfile:
+            tmpfile.write(test_data)
+            tmpfile.flush()
+            transfer.store_file_from_disk(key="test_key1", filepath=tmpfile.name)
+
+        client.putfo.assert_called()
+        notifier.object_created.assert_any_call(key="test_key1", size=len(test_data))
```

Comment:
I think this assertion is wrong. I'd assume (but you can prove me wrong) that we get 1 notification for each 1 object created, so this should assert that method 'object_created' is called once.

But I see that sftp store_file_from_disk calls store_file_object and both of them perform a notification, so there could be a bug there.

Reply:
Yes, we can use `assert_called_once_with()` everywhere to be more strict, no reason really to not do so.